invenio_app_ils/circulation/utils.py

```diff
@@ -70,6 +70,14 @@ def circulation_is_item_available(item_pid):
     return False
 
 
+def circulation_is_item_in_loanable_state(item_pid):
+    """Check if item is in a loanable state."""
+    item = Item.get_record_by_pid(item_pid)
+    if item:
+        return item["status"] not in ["MISSING", "IN_BINDING", "SCANNING"]
```

Comment:
There is already a method doing the same thing just above, `circulation_is_item_available`.
We should use that one probably, but I realised that it *could* also throw an exception.

Could you please use the already existing method and improve it with something like:
```
from invenio_pidstore.errors import PersistentIdentifierError

...
try:
    item = Item.get_record_by_pid(item_pid)
    return item["status"] == "LOANABLE"
except PersistentIdentifierError as ex:
    return False
```